.eslintrc.js

```diff
@@ -309,7 +298,6 @@ const stylisticIssuesRules = {
 	],
 	'no-multiple-empty-lines': ERROR,
 	'no-nested-ternary': ERROR,
-	'no-new-object': ERROR,
```

Comment:
I can see why we'd prefer to avoid it, but arrow functions returning new objects make this tricky.